timescaledb/how-to-guides/continuous-aggregates/drop-raw-data.md

```diff
@@ -21,3 +21,4 @@ raw data has subsequently been dropped.
 
 [api-drop-chunks]: /api/:currentVersion:/hypertable/drop_chunks
 [retention-aggregate]: /how-to-guides/data-retention-with-continuous-aggregates/
+[api-add-retention-policy]: /api/:currentVersion:/data-retention/add_retention_policy
```

Comment:
```suggestion
[api-add-retention-policy]: /api/add_retention_policy/
```

Reply:
Same as above, since we're going from `/timescaledb/latest` to `/api/latest`, we need to include the first to parts for the URL parser.